src/Symfony/Component/Serializer/Mapping/AttributeMetadataInterface.php

```diff
@@ -21,6 +21,9 @@
  * @internal
  *
  * @author Kévin Dunglas <dunglas@gmail.com>
+ *
+ * @method string[]       getSerializedNames() Gets all the serialized names per group ("*" being the base name applied to all groups).
```

Comment:
```suggestion
 * @method string[]       getSerializedNames() Gets all the serialized names per group.
```
I'm not sure if we should specify the `*` specifics here